homeassistant/components/media_player/strings.json

```
@@ -299,22 +299,22 @@
       "description": "Removes all items from the playlist."
     },
     "shuffle_set": {
-      "name": "Shuffle",
-      "description": "Playback mode that selects the media in randomized order.",
+      "name": "Shuffle set",
+      "description": "Enables or disables the shuffle mode.",
       "fields": {
         "shuffle": {
-          "name": "Shuffle",
-          "description": "Whether or not shuffle mode is enabled."
+          "name": "Shuffle mode",
+          "description": "Whether the media should be played in randomized order or not."
         }
       }
     },
     "repeat_set": {
-      "name": "Repeat",
-      "description": "Playback mode that plays the media in a loop.",
+      "name": "Repeat set",
+      "description": "Enables or disables the repeat mode.",
```

Repeat is not a `bool`:

```suggestion
      "description": "Sets the repeat mode.",
```